client/src/components/settings/toggle-radio-setting.tsx

```diff
@@ -41,27 +41,41 @@ export default function ToggleRadioSetting({
           {explain ? <p id={`desc${flagName}`}>{explain}</p> : null}
         </div>
         <div className='toggle-radio-group'>
-          <label htmlFor={firstRadioId} data-checked={flag}>
-            <input
-              id={firstRadioId}
-              type='radio'
-              {...(flag && { defaultChecked: true })}
-              {...(!flag && { onChange: toggleFlag })}
-              name={flagName}
-              value='1'
-            />
-            <span>{restProps.onLabel}</span>
+          <label
+            className='outer-label'
+            htmlFor={firstRadioId}
+            data-checked={flag}
+          >
+            <label className='custom-label'>
+              <input
+                id={firstRadioId}
+                type='radio'
+                {...(flag && { defaultChecked: true })}
+                {...(!flag && { onChange: toggleFlag })}
+                name={flagName}
+                value='1'
+              />
+              <span className='inner-circle'></span>
+            </label>
+            <span id='text-span'>{restProps.onLabel}</span>
           </label>
-          <label htmlFor={secondRadioId} data-checked={!flag}>
-            <input
-              id={secondRadioId}
-              type='radio'
-              {...(!flag && { defaultChecked: true })}
-              {...(flag && { onChange: toggleFlag })}
-              name={flagName}
-              value='2'
-            />
-            <span>{restProps.offLabel}</span>
+          <label
+            className='outer-label'
+            htmlFor={secondRadioId}
+            data-checked={!flag}
+          >
```

Comment:
Same here, can't be a `label`.

Reply:
You did change one of the `label`s to a `div` but now the `input` isn't accessible by a keyboard because it is being hidden by CSS. I really urge you to look closely at how these custom radio buttons are being implement on the other pages, for example [Introduction to Data Analysis](https://www.freecodecamp.org/learn/data-analysis-with-python/data-analysis-with-python-course/introduction-to-data-analysis). It is a very simple solution which moves the `input` off screen and uses just a couple of spans to create the radio button. I fear that your solution is starting to become too convoluted.

Reply:
Hey @bbsmooth, apologies for the convoluted approach. Have taken another approach of span (as you recommended) and the keys works fine now.

Reply:
No need to apologize.